### PR TITLE
feat:CHP-179: Add archive service

### PR DESCRIPTION
**Pull request by lquanx**
Issue #, if available:

Description of changes:

Checklist:

* [ ] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by Zambonilli — APPROVED** [review]

File: archiveServiceLambda/index.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { handleDdbToS3Event } from 'fhir-works-on-aws-persistence-ddb';
+
+exports.handler = async (event: any) => {
+    await handleDdbToS3Event(event);
```
Comment (Zambonilli) [inline archiveServiceLambda/index.ts:4]: can we open a new ticket to pass through a config value as opposed to the persistence-ddb parsing the environment var for archive config. downstream modules besides deployment should deal with configuration through the config interface so the implementation can change more easily.

Comment (lquanx) [reply archiveServiceLambda/index.ts:4]: That makes sense

**Review by nirojshrestha019 — APPROVED** [review]

File: archiveServiceLambda/index.ts (hunk 1 above)
Comment (lwarren01) [reply archiveServiceLambda/index.ts:4]: How do you see the Lamdbda function accessing the configuration? Passing it into the event handler function? Calling a method on the interface component? I am in the process of creating the ticket.

Comment (lquanx) [reply archiveServiceLambda/index.ts:4]: I think the Lambda function could access the configuration via environment variable, and pass the configuration to handleDdbToS3Event function. Like this
```
exports.handler = async (event: any) => {  
    // parse string format archive config into Map<string, number> format.
    const ttlsInSeconds = utility.parseArchiveConfig(process.env.ARCHIVE_CONFIG)
    await handleDdbToS3Event(event, ttlsInSeconds);
}
```

Comment (lwarren01) [reply archiveServiceLambda/index.ts:4]: We talked a little more about this while you were out. I have opened CHP-251. I think I captured what was discussed. Pleas take a look and followup in the Jira issue. Thanks.

**Pull request merged**